config/rapidez/sentry-vue.php

```diff
@@ -4,6 +4,7 @@
     // Configuration as defined in the docs: https://docs.sentry.io/platforms/javascript/guides/vue/configuration/
     'configuration' => [
         'enabled' => env('SENTRY_VUE_ENABLED', true),
+        'environment' => env('SENTRY_ENVIRONMENT', 'production'),
```

Comment:
Maybe listen to the app env by default?
```suggestion
        'environment' => env('SENTRY_ENVIRONMENT', env('APP_ENV', 'production')),
```